.github/workflows/ci.yml

```
@@ -137,8 +137,8 @@ jobs:
       - name: Test
         run: cargo test --no-fail-fast --all-targets --all-features --workspace
 
-      - name: Async-std Test
-        run: cargo test --no-fail-fast --all-targets --no-default-features  --features "async-std" --features "storage-all" --workspace
+      - name: Smol Test
+        run: cargo test --no-fail-fast --all-targets --no-default-features  --features "smol" --features "storage-all" --workspace
```

Unrelated, but I noticed that there is an extra space:
```suggestion
        run: cargo test --no-fail-fast --all-targets --no-default-features --features "smol" --features "storage-all" --workspace
```